Dart/src/com/jetbrains/lang/dart/ide/toolingDaemon/DartToolingDaemonService.kt

```diff
@@ -48,6 +48,7 @@ class DartToolingDaemonService private constructor(private val project: Project)
   private lateinit var webSocket: WebSocket
   var webSocketReady = false
     private set
+  var uri: String? = null
```

Comment:
I'll add `private set` to make sure that the value of this property can't be accidentally changed from elsewhere